standards/core/mix.md

```diff
@@ -0,0 +1,119 @@
+---
+title: WAKU-MIX
+name: Waku Mix
+editor: Prem Chaitanya Prathi <premprathi@proton.me>
+contributors:
+ - Akshaya Mani <akshaya@status.im>
+ - Hanno Cornelius <hanno@status.im>
+---
+
+# Abstract
+This document explains how [libp2p mix](https://rfc.vac.dev/vac/raw/mix/) protocol will be integrated into waku. This would provide higher anonymity for users publishing or querying for messages to/from the Waku network. 
+
+This document covers integration of mix with [lightpush](https://rfc.vac.dev/waku/standards/core/19/lightpush) and [store](https://rfc.vac.dev/waku/standards/core/13/store) protocols. It also covers the aspect of relay nodes acting as mix nodes.
+Both of these are request response based protocols that follow a service-user and service-provider model.
+
+Integration of gossipsub with mix is not covered in this document.
+
+## Background / Rationale / Motivation
+
+Waku protocols have weak sender/originator anonymity as explained in [Waku Privacy and Anonymity Analysis](https://vac.dev/rlog/wakuv2-relay-anon/).
+As of today, it is easy to analyze the network traffic and determine originator of messages published into the network or determine the topic interests by analyzing store queries requested by a user. Similar analysis is possible for topic subscriptions via [Filter](https://rfc.vac.dev/waku/standards/core/12/filter) protocol as well.
```

Comment:
It is better not to tie specifications to a temporary state of the art as with "as of today". I'd rather say something like: "without a further anonymisation layer, it is easy..."